utils/httpbin/httpbin.go

```diff
@@ -69,8 +69,9 @@ func delayHandler(w http.ResponseWriter, r *http.Request) {
 	case <-r.Context().Done():
 		t.Stop()
```

Comment:
Since we are here, shouldn't we also defer the timer stop?

Reply:
Don't mention it. It's a timer, not ticker.

Reply:
Sure, I'll add it.